pyfar/classes/filter.py

```diff
@@ -164,7 +164,7 @@ def coefficients(self):
         Get and set the coefficients of the filter.
 
         Refer to the
-        :py:mod:`filter concepts <pyfar._concepts.filter_concepts>` for use
```

Comment:
Were those links dead? I thought we already removed the concepts from main. Maybe not detected because thos is a parant class not included in the docs.

Reply:
Yes they were dead, I dont understand why its not returning a waring in these cases. I treid [nitpicky](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-nitpicky) but this just fail if the type of parameters cannot be found.

Reply:
Probably we did not get a warning since its not included in the docs and hence not touched by sphinx.